_posts/en/newsletters/2022-07-20-newsletter.md

```diff
@@ -0,0 +1,199 @@
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes several related discussions about providing
+a sustainable long term block reward for Bitcoin.  Also included are our
+regular sections with descriptions of new features for clients and
+services, announcements of new releases and release candidates, and
+summaries of notable changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Long-term block reward ongoing discussion:** continuing the
+  discussion about reliably paying for Proof of Work (PoW) as Bitcoin's
+  subsidy declines, two new threads were started on the Bitcoin-Dev
+  mailing list:
+
+    - [Tail emission is not inflationary][todd tail] starts with an argument by
+      Peter Todd that perpetually paying miners with newly created
+      bitcoins will not lead to the number of bitcoins in circulation
+      increasing forever.  Instead, he believes some bitcoins will be lost
+      every year and, eventually, the rate at which bitcoins are lost will
+      converge on the rate at which new bitcoins are produced, resulting in
+      a roughly stable number of coins in circulation.  He also notes
+      that adding a perpetual block subsidy to Bitcoin would be a hard
+      fork.  Quite a few people replied to his post and on a [thread
+      about it][talk tail] on BitcoinTalk; we will only attempt to summarize a
+      few replies we found most notable.
+
+        - *Hard fork not required:* Vjudeu [suggests][vjudeu sf] that a soft fork
+          can create new bitcoins by imbuing transaction outputs paying
+          zero satoshis with special meaning.  For example, when a node
+          opting into the fork sees a zero-sat output, it looks at
+          another part of the transaction for the actual value
+          transferred.  This creates two classes of bitcoins, but
+          presumably the soft fork would provide a mechanism to convert
+          legacy-bitcoins to modified-bitcoins.  Vjudeu notes the same
+          mechanism could also be used for privacy-enhancing bitcoin
+          amount blinding (e.g. using [confidential transactions][]).
+
+        - *No reason to believe perpetual issuance is sufficient*:
+          Anthony Towns [posts][towns pi] to the mailing list and Gregory
+          Maxwell [posts][maxwell pi] to BitcoinTalk that there's no reason to
+          believe that paying miners an amount of coins equal to the
+          average rate of lost coins will provide sufficient PoW
+          security, and that there are cases where it could overpay for
+          PoW security.  If a perpetual issuance can't guarantee security
+          and if it has a significant likelihood of producing additional
+          problems, it seems preferable to stick with a finite subsidy
+          that---while it also can't guarantee security---at least it
+          doesn't produce additional problems and is already
+          accepted by all Bitcoiners (implicitly or explicitly).
```

Comment:
apologies, this sentence might be 100% grammatically correct but confused me...?
```suggestion
          PoW security.  If a perpetual issuance can't guarantee security
          and if it has a significant likelihood of producing additional
          problems, it seems preferable to stick with a finite subsidy
          that---while also can't guarantee security---at least
          doesn't produce additional problems and is already
          accepted by all Bitcoiners (implicitly or explicitly).
```